java/code/src/com/redhat/rhn/domain/contentmgmt/ProjectSource.java

```diff
@@ -146,6 +146,20 @@ public ProjectSource(ContentProject project) {
         this.contentProject = project;
     }
 
+    /**
+     * Gets the source Label.
+     *
+     * @return source label
+     */
+    public abstract String sourceLabel();
+
+    /**
+     * Gets the source Type.
+     *
+     * @return source Type
+     */
+    public abstract Type sourceType();
```

Comment:
Same as before, according to default naming, this should be `getSourceType`, no?